hoomd/md/pytest/test_nlist.py

```diff
@@ -145,10 +145,60 @@ def test_pickling(simulation_factory, two_particle_snapshot_factory):
     pickling_check(nlist)
 
 
+def test_cell_properties(simulation_factory, lattice_snapshot_factory):
+    nlist = hoomd.md.nlist.Cell(buffer=0)
+    lj = hoomd.md.pair.LJ(nlist, default_r_cut=1.1)
+    lj.params[('A', 'A')] = dict(epsilon=1, sigma=1)
+    lj.params[('A', 'B')] = dict(epsilon=1, sigma=1)
+    lj.params[('B', 'B')] = dict(epsilon=1, sigma=1)
+    integrator = hoomd.md.Integrator(0.005)
+    integrator.forces.append(lj)
+    integrator.methods.append(
+        hoomd.md.methods.Langevin(hoomd.filter.All(), kT=1))
+
+    sim = simulation_factory(lattice_snapshot_factory(n=10))
+    sim.operations.integrator = integrator
+
+    sim.run(10)
+
+    assert nlist.num_builds == 10
+    assert nlist.shortest_rebuild == 1
+    dim = nlist._cpp_obj.getDim()
+    dim = nlist.dimensions
```

Comment:
Why define this twice?

Reply:
Good catch. I was finding the source of an `AttributeError`. Due to the way that Python processes `AttributeErrors`, the 2nd line made it hard to find the cause of the error, so I added the first. The first shouldn't be there, it was just for debugging.

Reply:
Ok, thanks